### PR TITLE
fix(SUP-14314): addressed the issue with the DualScreen container ove…

### PR DESCRIPTION
**Pull request by shalommeoded**
@eitanavgil please review PR

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @shalommeoded,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: modules/KalturaSupport/components/dualScreen/dualScreen.js [hunk 1]
```diff
@@ -105,7 +105,11 @@
                             }, 500);
                         }
                     }
-				} );
+                    if( _this.getPlayer().isMobileSkin() ) {
```
Comment (eitanavgil) [inline modules/KalturaSupport/components/dualScreen/dualScreen.js]: @shalommeoded 
I do not want to change the controllbar on any player that has this plugin (this is what your code will do) 
I rather do this with targeting the 'dualScreenControlBar' class and not controlBar.
Besides that - whats the point doing this in JQ and not as a regular well-targeted css? We have a .mwPlayerContainer.mobile.mobileSkin that we can use as a high-level umbrella.

Comment (shalommeoded) [reply modules/KalturaSupport/components/dualScreen/dualScreen.js]: @eitanavgil made the changes, please review.

File: modules/KalturaSupport/components/dualScreen/dualScreenControlBar.css [hunk 2]
```diff
@@ -218,6 +218,9 @@
     top: 0;
     left: 0;
 }
+.mwPlayerContainer.mobile.mobileSkin .controlBar.dualScreen{
```
Comment (eitanavgil) [inline modules/KalturaSupport/components/dualScreen/dualScreenControlBar.css]: @shalommeoded  this is nice and everything, yet you didn't delete the code that does the same thing

Comment (shalommeoded) [reply modules/KalturaSupport/components/dualScreen/dualScreenControlBar.css]: @eitanavgil Sorry, a noob mistake...

**Pull request merged**